ncard_app/models.py

```diff
@@ -278,3 +279,21 @@ class Meta:
         constraints = [
             models.UniqueConstraint(fields=['grant', 'investigator'], name='grantinvestigator_unique')
         ]
+
+
+class Students(models.Model): 
+    student_name = models.OneToOneField(Person, on_delete=models.CASCADE, related_name = 'person')
+    student_type = models.IntegerField('student type', )
+    primary_supervisor = models.CharField('primary supervisor', max_length=255, Blank=True)
+    supervisor2 = models.CharField('primary supervisor', max_length=255, Blank=True)
+    supervisor3 = models.CharField('primary supervisor', max_length=255, Blank=True)
+    title_topic = models.TextField('title topic', blank=True)
+    year_start = models.PositiveSmallIntegerField('year start')
+    year_end = models.PositiveSmallIntegerField('year end')
```

Comment:
year_start and year_end are optional in the Access database. Add the blank=True and null=True attributes